edit.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Product modify</title>
+</head>
+
+<body background="image/images.jpg">
+
+    <nav class="topnav">
+        <ul>
+            <li><a href="view_products.html">Products</a></li>
+            <li><a href="sales.html">Sales</a></li>
+            <li><a href="add.html">Add</a></li>
+            <li><a href="edit.html">Edit</a></li>
+            <li><a href="register.html">Register</a></li>
+            <li><a href="index.html">Logout</a></li>
+        </ul>
+    </nav>
+    <main class="boxed">
+
+        <table>
+            <thead>
+                <th>PRODUCT NAME</th>
+                <th>BRAND</th>
+                <th>EDIT</th>
+            </thead>
+            <tbody>
+                <td>Iphone 5</td>
+                <td>Phone</td>
+                <td><button type="submit">Edit</button><br><button type="submit">Delete</button> </td>
+            </tbody>
+            <tbody>
```

Comment:
Bad use of `tbody` tag. Expected to see it once and `tr` tags used the way you are using the `tbody`. See comments below